### PR TITLE
Add e2e tests

### PR DESCRIPTION
**Pull request by kvaps**
Add script that creates three VMs and installs Cozystack into them

### CODE REVIEW

File: hack/e2e.sh [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+done
+
+# Prepare networking
+for i in 1 2 3; do
+  ip link del cozy-srv$i || true
+  ip tuntap add dev cozy-srv$i mode tap
+  ip link set cozy-srv$i up
+  ip link set cozy-srv$i master cozy-br0
+done
+
```
Comment (themoriarti) [inline hack/e2e.sh:84]: To be able to go outside add:
```
echo 1 > /proc/sys/net/ipv4/ip_forward
iptables -t nat -A POSTROUTING -s 192.168.123.0/24 -j MASQUERADE
```

**Review by themoriarti — COMMENTED** [review]
After add MASQUERADE, it would be good to stop the virtual machines at the end of the test or add the possibility of configuration parameter for that.

File: hack/e2e.sh (hunk 1 above)
Comment (kvaps) [reply hack/e2e.sh:84]: I have to add check to test for this

Comment (kvaps) [reply hack/e2e.sh:84]: Thanks, just added this to tests

**Review by themoriarti — APPROVED** [review]
Now looks good.

**Pull request merged**